chart/values.yaml

```diff
@@ -56,6 +56,7 @@ controller:
       enabled: true # enable the provider
       ipRangesURL: "https://ip-ranges.datadoghq.com/" # URL to the IP ranges file (format must be the expected one, defaults is the public file provided by the cloud provider)
   defaultDuration: 1h # default spec.duration for a disruption with none specified
+  maxDuration: # maximum spec.duration for a disruption (example: 1h)
```

Comment:
```suggestion
  maxDuration: 1h # maximum spec.duration for a disruption (example: 1h)
```

Reply:
The `maxDuration` will be defined on all environments if we defined a default value here. Is it a problem?

Reply:
> on all environments 

what do you mean? Also I'd argue for a larger default max than 1h

Reply:
I mean do we want by default to have a `maxDuration` or not? 
**IMO** it makes sense to have a default `maxDuration`.